VERSION

```diff
@@ -0,0 +1 @@
+2022-02-10
```

Comment:
I think we should make this `20220210` so that `latest` will honor these versions. I think (not 100%) thta `2022-02-10` would not be `latest`

Reply:
Still, that is the last time we released the builder.